linux-support.md

```diff
@@ -41,30 +44,16 @@ ldd (Ubuntu GLIBC 2.35-0ubuntu3.1) 2.35
 
 ## OpenSSL compatibility
 
-Microsoft builds support both OpenSSL 1.x and 3.x and can be run on distributions with either version of this package. For example, Ubuntu 22.04 only includes OpenSSL 3 in its official package archive.
-
-Microsoft builds will generally load the highest OpenSSL version it finds, but can be configured to use a specific version.
-
-## Red Hat Enterprise Linux Family support
-
-Microsoft builds support multiple Red Hat versions. New .NET versions will typically only be supported on RHEL era distributions in active support.
-
-- RHEL 7 era distributions are considered in maintenance.
-- RHEL 8 era distributions are considered in active support.
-- RHEL 9 era distributions are considered in active support.
-
-Red Hat family distributions include: AlmaLinux, CentOS, Oracle Linux, Red Hat Enterprise Linux, and Rocky Linux.
-
-## Red Hat Commercial Support
+Portable builds support both OpenSSL 1.x and 3.x and can be run on distributions with either version of OpenSSL. For example, Ubuntu 22.04 only includes OpenSSL 3 in its official package archive.
 
-Red Hat supports .NET via [Red Hat Enterprise Linux](http://redhatloves.net/), per the following.
+The highest OpenSSL version is loaded by default, but can be [configured to use a specific version](https://github.com/dotnet/runtime/issues/79153#issuecomment-1335476471).
```

Comment:
```suggestion
The highest OpenSSL version is loaded by default, but it can be [configured to use a specific version](https://github.com/dotnet/runtime/issues/79153#issuecomment-1335476471).
```